Dashboard/Views/MainWindowView.axaml

```diff
@@ -0,0 +1,60 @@
+﻿<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:views="clr-namespace:Dashboard.Views"
+        xmlns:vm="clr-namespace:Dashboard.ViewModels"
+        xmlns:models="clr-namespace:Dashboard.Models"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="Dashboard.Views.MainWindowView"
+        x:DataType="vm:MainWindowViewModel"
+        Title="MainWindow"
+        Foreground="Black"
+        Background="#c4c4c4">
+        <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:MainWindowViewModel />
+    </Design.DataContext>
+    
+    <Grid RowDefinitions="Auto, *, Auto" >
+        
+        <Grid Grid.Row="0" Height="100" >
+                <Image Source="/Assets/nuvotion_logo.png" Width="100" Height="100" HorizontalAlignment="Right"
+                       VerticalAlignment="Top" RenderTransformOrigin="0.5,0.5">
+                        <Image.RenderTransform>
+                            <RotateTransform Angle="90" />
+                        </Image.RenderTransform>
+                    </Image>
+                    <TextBlock> Nuvotion Motorsports</TextBlock>
```

Comment:
This looks quite small in the corner, can we vertically centre it and make it a bit bigger? Alternatively, could we consider making this something like "FSAE Scrutineering Tool" -- need Nick's input on this one

Reply:
19b1f940a5ff9e1743d7213780f9944f81b4fd27